### PR TITLE
Added services description document

### PR DESCRIPTION
**Pull request by ptimmons**
As requested by Andy Epstein and Brad Jones (JNPR legal).

### CODE REVIEW

File: docs/about_services_description.md [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+title: Services Description
+sidebar_label: Services Description
+description: 'This document describes the maintenance and support services available to 128 Technology's end customers.'
+---
+
+## Introduction and Eligibility
+
+This Services Description Document (“SDD”) describes the maintenance and support services (“Services”) available to licensed subscribers (“Customers”) of 128 Technology software products (“Software”). 
+
+The Services are subject to the terms of this SDD and the 128 Technology End User License Agreement or other written agreement covering within its scope the terms and conditions under which Customer has purchased a license subscription to the Software (“Subscription Contract”).
```
Comment (Chr1st0ph3rTurn3r) [inline docs/about_services_description.md:11]: Globally, Customer should be prefaced with an article - the Customer, a Customer. Or, it should be changed to Customers, as indicated in the first paragraph. 
There are several instances where this must be looked at individually, based on sentence structure.

File: docs/about_services_description.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+### Software Releases
+
+During the Service Period, 128 Technology shall make available to Customer the Updates (defined below) of the Software covered under a Subscription Contract, but solely for Customer’s use as a replacement of a different release of the Software duly licensed to such Customer. Any such Update shall be subject to the terms and conditions set forth below:
+
+I. Rights in Updates. An Update downloaded by or otherwise delivered to Customer under a Subscription Contract may be used only to replace Software licensed to Customer and only if Customer agrees not to use the Update until it first discontinues use of the replaced Software and makes it unavailable for any use, other than for archival use. Once an Update is installed and placed in use and all copies of the replaced release of the Software is removed from all use, the Customer’s rights in the Update will be subject to the same terms, conditions, and restrictions as the Software originally licensed to Customer, as set forth in the applicable Subscription Contract.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/about_services_description.md:31]: "Software licensed to Customer and only if Customer agrees not..." 
In this case customer should be prefaced with an article - "licensed to the Customer, and only if the Customer agrees..."

File: sidebars.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ module.exports = {
       "about_supported_platforms",
       "about_certified_platforms",
       "about_security_policy",
+      "about_services_description"
```
Comment (MichaelBaj) [inline sidebars.js:10]: missing comma at the end of this list, causing the "build" to fail

File: docs/about_services_description.md (hunk 2 above)
Comment (MichaelBaj) [inline docs/about_services_description.md:31]: I'm presuming this statement came from legal, so I don't imagine we should change it. Maybe I'm being too technical, but older versions of the software will exist for the purposes of rolling back.

File: docs/about_services_description.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+Customer may from time to time submit requests or suggestions for feature or functionality enhancements that would require future development. 128 Technology may review and discuss such requests with Customer on a periodic basis, as mutually determined by the parties. 128 Technology shall have no obligation to develop or deliver any enhancements.
+
+**Escalation**
+
+Customer and 128 Technology may establish and periodically update an escalation list of management employees responsible for the support of the Software. Use of such list will be subject to compliance with 128 Technology’s guidelines and policies, including those applicable to privacy.
+
+**Service Level Targets**
+
+The response times and other service levels described in this Schedule 1 are targets or objectives that 128 Technology strives to achieve consistently in providing Services to Customers. A failure to meet any one or more of the service level targets will not create any liability on the part of 128 Technology, nor a right to any credits or to withhold payment on the part of Customers.
```
Comment (Chr1st0ph3rTurn3r) [inline docs/about_services_description.md:180]: "described in this Schedule 1 are..." 
This is a bit funky. "described in Schedule 1 are..." or "described in this Schedule are..."?

**Review by Chr1st0ph3rTurn3r — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/about_services_description.md (hunk 1 above)
Comment (ptimmons) [reply docs/about_services_description.md:11]: This document was copied and pasted verbatim from Juniper corporate legal and Andy Epstein. Unless this is a hard requirement of yours I suggest we leave it as-is.

File: docs/about_services_description.md (hunk 2 above)
Comment (ptimmons) [reply docs/about_services_description.md:31]: This document was copied and pasted verbatim from Juniper corporate legal and Andy Epstein. Unless this is a hard requirement of yours I suggest we leave it as-is.

File: docs/about_services_description.md (hunk 4 above)
Comment (ptimmons) [reply docs/about_services_description.md:180]: This document was copied and pasted verbatim from Juniper corporate legal and Andy Epstein. Unless this is a hard requirement of yours I suggest we leave it as-is.

File: docs/about_services_description.md (hunk 2 above)
Comment (ptimmons) [reply docs/about_services_description.md:31]: Correct, it came from legal. I suggest we leave it as-is.

File: sidebars.js (hunk 3 above)
Comment (ptimmons) [reply sidebars.js:10]: Fixed.

**Review by MichaelBaj — APPROVED** [review]

**Comment by MichaelBaj** [comment]
The build failed due to

```
Creating an optimized production build...
Error: Error while parsing markdown front matter.
This can happen if you use special characteres like : in frontmatter values (try using "" around that value)
can not read a block mapping entry; a multiline key may not be an implicit key at line 5, column 1:
```

I suspect this is because you used a single quote instead of a double quote for the description.

**Pull request merged**